tests/unit/admin/test-class-wc-rest-payments-settings-controller.php

```diff
@@ -68,9 +59,8 @@ public function setUp() {
 		$token_service            = new WC_Payments_Token_Service( $this->mock_api_client, $customer_service );
 		$action_scheduler_service = new WC_Payments_Action_Scheduler_Service( $this->mock_api_client );
 
-		$this->gateway                 = new WC_Payment_Gateway_WCPay( $this->mock_api_client, $account, $customer_service, $token_service, $action_scheduler_service );
-		$this->digital_wallets_gateway = new Digital_Wallets_Payment_Gateway( $this->mock_api_client, $account, $customer_service, $token_service, $action_scheduler_service );
```

Comment:
`Digital_Wallets_Payment_Gateway` is no longer a dependency on `WC_REST_Payments_Settings_Controller`